### PR TITLE
Add client passthrough documentation

### PR DESCRIPTION
**Pull request by victortoso**
At the moment, only USB Redireciton with usbredir is supported.
Added in https://github.com/kubevirt/kubevirt/pull/4089

### CODE REVIEW

File: docs/operations/client_passthrough.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+### Using Bus and Device address
+
+Redirecting the integrated camera
+```
+virtctl usbredir 01-03 vmi-name
+```
+
+### Requirements
```
Comment (fabiand) [inline docs/operations/client_passthrough.md:46]: Please clarify if these are reqiurements for the virttl/client or serve side

File: docs/operations/client_passthrough.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+#### usbredirect
+
```
Comment (fabiand) [inline docs/operations/client_passthrough.md:49]: Do you mind addding a small sentence about what usbredir is daemon used by virtctl to …

**Review by fabiand — APPROVED** [review]
Awesome. Thanks, a few nits

**Comment by fabiand** [comment]
/approve
With the trust that the issues will be addressed.

File: docs/operations/client_passthrough.md (hunk 1 above)
Comment (victortoso) [reply docs/operations/client_passthrough.md:46]: The requirements are for `virtctl usbredir` to function properly client side. Cluster side only requires the VMI's device configuration.

File: docs/operations/client_passthrough.md (hunk 2 above)
Comment (victortoso) [reply docs/operations/client_passthrough.md:49]: Sure, I'll clarify what `usbredirect` does when `virtctl usbredir` command is used.

**Comment by victortoso** [comment]
> With the trust that the issues will be addressed.

Sure, I'll be fixing those in a follow up MR. Thanks for the quick review

**Pull request merged**